app/models/grda_warehouse/warehouse_reports/dashboard/entered.rb

```diff
@@ -68,16 +70,16 @@ def run!
       # ensure that the counts are in the same order as the labels
       @labels.each do |project_type_sym, _|
         @buckets.each do |project_type, bucket|
-          project_type_key = ::HudUtility2024.project_type_brief(project_type).downcase.to_sym
-          if project_type_sym == project_type_key
-            bucket.each do |group_key, client_count|
-              @data[group_key][:data] << client_count
-            end
+          project_type_key = ::HudUtilityCurrent.project_type_brief(project_type).downcase.to_sym
+          next unless project_type_sym == project_type_key
+
+          bucket.each do |group_key, client_count|
+            @data[group_key][:data] << client_count
           end
         end
       end
 
-      data = {
+      data = { # rubocop:disable Lint/UselessAssignment
```

Comment:
is there some reason to keep the assignment?
```suggestion
      {
```